ipaserver/install/ca.py

```diff
@@ -177,14 +177,14 @@ def install_check(standalone, replica_config, options):
     if standalone:
         dirname = dsinstance.config_dirname(
             installutils.realm_to_serverid(realm_name))
-        cadb = certs.CertDB(realm_name, subject_base=options._subject_base)
+        cadb = certs.CertDB(realm_name, paths.HTTPD_ALIAS_DIR,
+                            subject_base=options._subject_base)
         dsdb = certs.CertDB(
             realm_name, nssdir=dirname, subject_base=options._subject_base)
 
         for db in (cadb, dsdb):
             for nickname, _trust_flags in db.list_certs():
-                if nickname in (certdb.get_ca_nickname(realm_name),
-                                'ipaCert'):
```

Comment:
Few lines below is a subject name check, you should remove the RA agent from there as well.